canonical-web-and-design/maas-ui
# feat(ui): add sorting to LXD project list

## Done

- added sorting to LXD project list, which simultaneously fixed a bug where the order of the LXD pods depended on when they were loaded into state

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the KVM list and check that you can sort LXD groups as well as the projects within the groups

## Fixes

Fixes canonical-web-and-design/MAAS-squad#2406
